Formula/crowdin.rb

```diff
@@ -1,9 +1,8 @@
 class Crowdin < Formula
   desc "Command-line tool that allows to manage your resources with crowdin.com"
   homepage "https://support.crowdin.com/cli-tool/"
-  url "https://downloads.crowdin.com/cli/v2/crowdin-cli-2.0.31.zip"
-  sha256 "93defe16706783e92cbe3b32e528e495ddffa9e2a68471c3b70a2eb6c487e245"
-  revision 1
+  url "https://github.com/crowdin/crowdin-cli/releases/download/3.1.15/crowdin-cli.zip"
```

Comment:
It's there no tar.gz?

Reply:
it is a java formula, I dont expect a tgz file has the executable binary though.